src/assets/simp/build/simp.spec

```diff
@@ -76,7 +76,6 @@ Requires: pupmod-simp-rsync >= 6.1.1, pupmod-simp-rsync < 7.0.0
 Requires: pupmod-simp-rsyslog >= 7.3.1, pupmod-simp-rsyslog < 8.0.0
 Requires: pupmod-simp-selinux >= 2.3.1, pupmod-simp-selinux < 3.0.0
 Requires: pupmod-simp-simp >= 4.6.0, pupmod-simp-simp < 5.0.0
-Requires: pupmod-simp-simpcat >= 6.0.3, pupmod-simp-simpcat < 7.0.0
```

Comment:
You need an obsoletes in the simp package to ensure pupmod-simp-simpcat is removed upon upgrade.  Otherwise it will remain on the system and may cause dependency issues down the road.